includes/class-scd-ext-drip-email.php

```diff
@@ -399,14 +399,27 @@ private function get_ordered_courses_and_lessons( $lessons ) {
 
 		// Sort list of lessons for each course.
 		foreach ( $courses_and_lessons as $course_id => $lesson_data_items ) {
-			// Set the current order as the default just in case the course lesson order is not set
+
```

Comment:
We've been slowly removing the blank lines padding the inside of control structures. It exists all over the plugin still.

(Also in lines 410+412)

Reply:
I am not sure if we should have rules about blank lines since I think that there can't be a guide which fits in every possible case. In most cases we should do what 'feels' right.

For example for what you suggest it might be ok in some of the cases but it could make the code hard to read in others. For example the code would start to feel cluttered if you had an if/else with 2-3 assignments in each branch and no blank lines or if you had nested if/else statements.

For the specifc case I can see no harm removing the blank line in 402 however removing lines in 410 and 412 I think it makes it a bit harder to read as the if/else together with the second if inside it makes it a bit too compact. What do you think?

Reply:
The problem with inconsistency in code styles is that it's quite subjective. While you might leave these blank lines in, someone else would likely come in later and remove them.

I would also argue that multiple `if/else` and nested `if` statements are themselves not very readable and can often be refactored to be more succinct and improve clarity.

Reply:
I agree that many things in code styles are subjective. However what I am trying to say is that by introducing such a rule we actually lose clarity, at least for some of the cases, which defeats purpose.

If this rule has be decided already and is not subject to change, I am fine with removing the lines however do we have such decisions documented somewhere so I can be aware of them?

Reply:
> do we have such decisions documented somewhere so I can be aware of them?

Not to my knowledge.